scripts/deploy/deploy_testflight.sh

```
@@ -120,25 +120,25 @@ MATCH_GIT_URL=$(grep -E "^apple\.match\.git\.url=" gradle/fork.properties 2>/dev
 
 # Validate App Store Connect API key configuration
 if [ -z "$APPSTORE_KEY_ID" ] || [ -z "$APPSTORE_ISSUER_ID" ]; then
     print_error "App Store Connect API credentials not configured"
-    print_info "Ensure secrets/apple/appstore/key_id and secrets/apple/appstore/issuer_id exist"
+    print_info "Ensure secrets/live/apple/appstore/key_id and secrets/live/apple/appstore/issuer_id exist"
     exit 1
 fi
 
 # Load Match password
-export MATCH_PASSWORD=$(cat secrets/apple/match/.match_password)
+export MATCH_PASSWORD=$(cat secrets/live/apple/match/.match_password)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)